### PR TITLE
btrfs-progs: upgrade to 6.8

### PR DESCRIPTION
**Pull request by BettyRain**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
btrfs-progs: upgrade to 6.8

###### Change Log  

- Upgrade btrfs-progs to 6.8

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Containerized build
- Buddy build id: [538765](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=538765&view=results)

### CODE REVIEW

File: SPECS/btrfs-progs/btrfs-progs.spec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Name:       btrfs-progs
-Version:    5.16
-Release:    102%{?dist}
+Version:	6.8
```
Comment (BettyRain) [inline SPECS/btrfs-progs/btrfs-progs.spec:2]: Idk why it looks weird here, in my VS code it looks perfect:
image

Comment (anphel31) [reply SPECS/btrfs-progs/btrfs-progs.spec:2]: Try turning on "Unicode Highlight: Invisible Characters" setting in VsCode. It looks like you've got tab characters that should be replaced.

**Review by anphel31 — APPROVED** [review]

File: SPECS/btrfs-progs/btrfs-progs.spec (hunk 1 above)
Comment (BettyRain) [reply SPECS/btrfs-progs/btrfs-progs.spec]: Fixed!
Thank you for a hint

File: SPECS/btrfs-progs/btrfs-progs.spec [hunk 2]
```diff
@@ -41,13 +43,14 @@ btrfs filesystem-specific programs.
 
 %build
 ./autogen.sh
-%configure \
-	--disable-zstd
-make DISABLE_DOCUMENTATION=1 %{?_smp_mflags}
+%configure CFLAGS="%{optflags} -fno-strict-aliasing" --with-crypto=libgcrypt --disable-python
```
Comment (jslobodzian) [inline SPECS/btrfs-progs/btrfs-progs.spec:46]: We dropped the disable-zstd.

if that option means “use system zstd” (not the embedded one) AND the option still exists, then can we put it back to be explicit that we are not using the built in version…. Helps with CVE disputes, and guarantees we’re not accidentally using some busted internal version

**Pull request merged**